printer_mods/davidauvergne/README.txt

```diff
@@ -1,5 +1,9 @@
 A remake of ryandam's Mod for a centered cable management.
-You can find the original here:
-https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/ryandam/Cable_management_duct
+
+You can find the original file here:
+(https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/ryandam/Cable_management_duct)

```

Comment:
Please make this a relative link instead of absolute (../../ryandam/cable_management_duct)

Reply:
Relative link is done. Sorry for these errors. Thank you :)